packages/kbn-plugin-helpers/tasks/build/create_build.js

```diff
@@ -119,7 +124,31 @@ module.exports = function createBuild(
       );
     })
     .then(function() {
-      const buildConfigPath = join(buildRoot, 'tsconfig.json');
+      if (!plugin.styleSheetToCompile) {
+        return;
+      }
+
+      const file = path.join(plugin.root, plugin.styleSheetToCompile);
```

Comment:
`path.join()` won't translate the `/` to `\` in `plugin.styleSheetToComplie` on windows. Use `path.resolve()` instead.

Reply:
(keeping track of which path segments might have path separators in them is why I basically always use `path.resolve()` instead of join)